lib/ts/types.ts

```diff
@@ -110,3 +110,32 @@ export type RecipeInterface = {
 
     signOut: (input: { userContext: any }) => Promise<void>;
 };
+
```

Comment:
should the recipe interface for sessions also have a `getGlobalClaimValidators` function which the user can override? Just like how it's done on the backend?

Reply:
@porcellus missed this?